### ISSUE TITLE
[Spell] Warrior block does not count well

### ISSUE THREAD

**Issue by Tyrannize999** [SunstriderEmu/BugTracker#1839]
**Describe the bug**
Currently blocking damage is counting wrong because it isn't block right the number what the stats says. My block stops 61 incoming dmg and when I fight with mobs they hit me around 55-75 dmg. If I use my block then should be blocking 61 dmg so dat dmg should be max 14 but it isn't. It says the mob hit me for 45 dmg and blocked 61. This isn't a critical strike was a normal hit. I tried with other mobs, same thing happened. 

**To Reproduce**
1. Cast Shield Block when you fight.
2. Blocking the damage is incorrect

**Expected behavior**
When you scroll over your blocking stat in defenses it'll say something like your block stops 61 damage. If you get a hit - say 100 dmg - you'll block 61 of it and only get hit for 39dmg.
**Screenshots/videos**
![WoWScrnShot_020920_120625](https://user-images.githubusercontent.com/60829714/74101108-829f0c00-4b36-11ea-8506-bc35d2325556.jpg)

![WoWScrnShot_020920_122202](https://user-images.githubusercontent.com/60829714/74101140-e6c1d000-4b36-11ea-9e76-3cdbbad059ea.jpg)

![WoWScrnShot_020920_173539](https://user-images.githubusercontent.com/60829714/74106533-13421000-4b68-11ea-8f9a-7aa6e6e4563b.jpg)

**Comment by Jerrythemagicdog** [comment]
Tagar Spinebreaker can currently hit between 756 and 1436 damage (between armor reductions). 
Umbraglow Stinger can hit between 131 and 183 damage.
Please provide screenshots of your Defenses character panel section with the block value and screenshots of combat log providing proof that the amount blocked is indeed incorrect.

**Comment by Tyrannize999** [comment]
Yea you are right about currently dmg from those mobs. I have 17120 armor (61.86% damage reduce) and my block stops 345 dmg. So Tagar should be damage on me around 289-548. He is hitting me from 200 and his best attack was 350 but it's okay. My problem is he only hits me harder if I'm using the Shield Block spell. **So actually I misspelled the problem, my mistake sorry.** The damage number what I block is correct, but the damage from mob is not. I mean like I said, the mob only hit with high damage if I use Shield Block. And I tried with a lot of mobs, same happened. Don't know it's understandable what I want to say. So he should be able to attack me with that damage, but why just only if I'm blocking? When he has a success attack 10 from 10 is low dmg. So what I want to say, what is the point of blocking if he hits harder after I used the spell.

I can give you only my defense stats picture. From combat log I have only the Damage what I've taken, but there is no number of counting how many attack did Tagar on me.
![WoWScrnShot_021020_115625](https://user-images.githubusercontent.com/60829714/74145198-85653400-4bfe-11ea-94cc-693d86005ddd.jpg)
